packages/feedback/stories/FeedbackWithModal.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { StoryObj } from '@storybook/react';
+
+import Feedback from '../src/Feedback';
+import FeedbackForm from '../src/FeedbackForm';
+// import README from '../README.md';
+
+export default {
+  title: 'Components/Feedback/FeedbackWithModal',
+  components: Feedback,
```

Comment:
I think there was some confusion here. Feedback with modal still is based on the feedback component, so should stay with the other feedback stories. FeedbackForm is different since it's a separate componnet.